app/models/enterprise.rb

```diff
@@ -198,10 +188,6 @@ def self.distinct_count
     count(distinct: true)
   end
 
-  def activated?
-    confirmed_at.present? && sells != 'unspecified'
```

Comment:
Is it even possible to create an enterprise without specifying the `sells` attribute?